splinepy/microstructure/tiles/hollow_octagon_extrude.py

```
@@ -250,351 +273,982 @@ def closing_tile(
+                        [
+                            v_h_void + v_one_half,
+                            -v_inner_c_h + v_one_half,
+                            v_one,
+                        ],
+                        [v_one, -v_outer_c_h + v_one_half, v_one],
+                    ]
+                )
 
-        spline_list.append(_Bezier(degrees=[1, 1], control_points=top))
+            for control_points in [
+                right,
```

Pylint (3.3.7)

[error] Possibly using variable 'right' before assignment

(E0606)